Sources/XcbeautifyLib/Renderers/TeamCityRenderer.swift

```diff
@@ -0,0 +1,262 @@
+import Colorizer
+import Foundation
+
+struct TeamCityRenderer: OutputRendering {
```

Comment:
There's a lot of new formatting logic in this PR. Similar to the `TerminalRenderer` and `GitHubActionsRenderer`, can you write tests for the expected raw inputs and formatted outputs?

Soon, I'm planning to merge a large change to formatting, so tests against TeamCity-related formatting will be important then, and as we maintain this library.

Reply:
Yep, sure! Can you rerun GitHub Actions to check PR?